.travis.yml

```diff
@@ -2,6 +2,8 @@ language: node_js
 node_js:
   - '7'
 env:
+  global:
+    - COMMIT=${TRAVIS_COMMIT::8}
```

Comment:
I don't think it changed the matrix, https://travis-ci.org/primer/primer-css/builds/251305487 

Looks like it just included a `$COMMIT` variable in all the builds